.gitattributes

```diff
@@ -1 +1,2 @@
 assets/** filter=lfs diff=lfs merge=lfs -text
+*.pdf filter=lfs diff=lfs merge=lfs -text
```

Comment:
This shouldn't need to be here, should be covered by the first line. Did you add this?